apps/software/lib/software/model/storage_drive.ex

```diff
@@ -16,6 +16,7 @@ defmodule Helix.Software.Model.StorageDrive do
   @creation_fields ~w/storage_id/a
 
   @primary_key false
+  @ecto_autogenerate {:drive_id, {PK, :pk_for, [__MODULE__]}}
```

Comment:
This is wrong (and the model was wrong before); `drive_id` is actually the ID of the HDD linked to the storage